File: AD/Models/Advertisement.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.ComponentModel.DataAnnotations.Schema;

namespace AD.Models
{
    public class Advertisement
    {
        public int Id { get; set; }

        public string Title { get; set; }

        public string Description { get; set; }

        public int Price { get; set; }

        public virtual List<Resource> Resources { get; set; }

        [ForeignKey("User")]
        public int UserId { get; set; }

        public virtual User User { get; set; }

        [ForeignKey("Category")]
        public int CategoryId { get; set; }

        public virtual Category Category { get; set; }

        [ForeignKey("Type")]
        public int TypeId { get; set; }

        public virtual AdvType Type{ get; set; }
    }
}

File: Advertisements.Web/UI/src/app/models/advertisement.ts
<<<<<<< HEAD
export class Advertisement{
  Id : number;
  Description : string;
  Myfield : string;
  Price : number;
  User : string;
  UserId : number;
  Category : string;
  CategoryId : number;
  Type : string;
=======
import {Resource} from './resource';

export class Advertisement{
  Id : number;
  Title : string;
  Description : string;
  Price : number;
  Resources : Resource[];
  ApplicationUserId : string;
  CategoryId : number;
>>>>>>> 4e6b888bd9e10a264d0007078d4833eef042529d
  TypeId :number;
}